src/applications/vaos/project-cheetah/index.jsx

```diff
@@ -56,6 +73,10 @@ export function NewBookingSection({ featureProjectCheetah }) {
 
   return (
     <FormLayout>
+      {(windowsStatus === FETCH_STATUS.loading ||
+        windowsStatus === FETCH_STATUS.notStarted) && (
+        <LoadingIndicator message="Checking COVID-19 vaccine availability" />
+      )}
```

Comment:
I think we probably want to hide the routes below while the availability check is loading

Reply:
Done